controllers/FavoriteCommentController.js

```diff
@@ -0,0 +1,197 @@
+import { FavoriteComment, Comment } from '../database/models';
+
+/**
+ *
+ * a class controller for adding like and dislike to the comment on article
+ * it includes a function to get all likes
```

Comment:
Can we delete this comment entirely? Like I said before, comments are a double edged sword which means they can be good but also irritating, this statement can actually be added on line 10....Like so 

```
 /**
  * A class controller for adding like and dislike to the comment on an article
  * it includes a function to get all likes
  *
  * @author Daniel
  * @param {*} req
  * @param {*} res
  * @returns {*} object
  */
```